packages/immutadot-parser/package.json

```diff
@@ -19,19 +19,23 @@
     "Hugo WOOD (https://github.com/hgwood)"
   ],
   "devDependencies": {
-    "babel-cli": "~6.26.0",
-    "cross-env": "~5.1.4",
+    "@types/jest": "^23.3.12",
     "eslint": "~4.19.1",
-    "jest": "~21.2.1",
+    "jest": "~23.6.0",
     "jsdoc": "~3.5.5",
-    "lerna": "~2.11.0"
+    "lerna": "~3.10.2",
+    "ts-jest": "^23.10.5",
+    "tslint": "^5.12.0"
   },
   "scripts": {
-    "build": "cross-env BABEL_ENV=production babel src -d .",
-    "lint": "eslint src",
-    "test": "jest -c ../../config/jest.js",
+    "build": "tsc -b config/tsconfig.json",
+    "lint": "tslint -c config/tslint.json src/**/*",
+    "test": "jest -c config/jest.js",
     "test:coverage": "jest -c ../../config/jest.js --maxWorkers=2 --coverage",
     "docs": "jsdoc -c ../../config/jsdoc.json",
     "docs:private": "jsdoc -c ../../config/jsdoc.json -p"
+  },
+  "dependencies": {
+    "typescript": "^3.2.2"
```

Comment:
typescript should probably be in devDependencies, shouldn't it?

Reply:
Done